pynicotine/shares.py

```diff
@@ -104,8 +104,7 @@ def run(self):
                 _new_mtimes, new_files, _new_streams = self.rescan_dirs("buddy", new_mtimes, new_files,
                                                                         new_streams, self.rebuild)
 
-                self.queue.put((_("%(num)s folders found after rescan"), {"num": len(new_files)}, None))
-                self.queue.put((_("Finished rescanning shares"), None, None))
+                self.queue.put((_("%(num)s folders found after rescan, finished."), {"num": len(new_files)}, None))
```

Comment:
We don't include full stops in log messages, unless there are multiple sentences.

Reply:
This isn't really a very good sentence anyway. Maybe the word ", finished" isn't needed at all.

Reply:
It can probably be removed.

Reply:
```
[2021-11-17 22:39:53] Connecting to server.slsknet.org:2242
[2021-11-17 22:39:53] Rescanning shares…
[2021-11-17 22:39:53] 25 folders found before rescan, rebuilding…
[2021-11-17 22:39:53] Error while scanning folder /media/user/USER/Music PLAY: [Errno 2] No such file or directory: '/media/user/USER/Music PLAY'
[2021-11-17 22:39:53] Rescan progress: 4 %
```
...
```
[2021-11-17 22:39:53] Rescan progress: 96 %
[2021-11-17 22:39:53] Rescan progress: 100 %
[2021-11-17 22:39:53] Rescan complete: 25 shared folders
[2021-11-17 22:39:53] Connected to server server.slsknet.org:2242, logging in…

```
![image](https://user-images.githubusercontent.com/88614182/142294684-d05da7e4-0607-4c50-8fc4-4e1bbc2d827c.png)